pkg/operator/ceph/file/filesystem.go

```diff
@@ -232,7 +232,7 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 
 	// create the filesystem ('fs new' needs to be forced in order to reuse pre-existing pools)
 	// if only one pool is created new it wont work (to avoid inconsistencies).
-	if err := client.CreateFilesystem(context, clusterName, f.Name, f.metadataPool.Name, dataPoolNames, !poolsCreated); err != nil {
+	if err := client.CreateFilesystem(cephVersion, context, clusterName, f.Name, f.metadataPool.Name, dataPoolNames, !poolsCreated); err != nil {
```

Comment:
Depending on what linter you are using, a context should come first in the argument list.
See https://github.com/golang/lint/commit/3390df4df2787994aea98de825b964ac7944b817 ;)

Reply:
Sounds good to me, will change that thanks!